app/models/group.rb

```diff
@@ -61,6 +61,16 @@ def all_people
     Person.all_in_groups(subtree_ids)
   end
 
+  def completion_score
+    people = all_people
+    if people.blank?
+      0
+    else
+      total_score = people.map(&:completion_score).inject(0) {|s, total| total + s }
```

Comment:
Isn't it possible to do just one loop instead of `map` and `inject`? I think something like this could work:

```ruby
people.inject(0) { |person, total| total + person.completion_score }
```